ste/mgr-JobPartTransferMgr.go

```diff
@@ -329,7 +332,6 @@ func (jptm *jobPartTransferMgr) Info() *TransferInfo {
 	}
 
 	if versionID != "" {
-		versionID = "versionId=" + versionID
```

Comment:
line 340 and 342 below, you need to make it 

+= "&versionId=" + versionID
and then the other line
= "versionId=" + versionID


similar change with snapshot

Reply:
added now!